.github/workflows/build_documentation.yml

```
@@ -13,6 +13,7 @@ jobs:
     with:
       commit_sha: ${{ github.sha }}
       package: diffusers
+      notebook_folder: diffusers
```

```suggestion
      notebook_folder: diffusers_doc
```
To be consistent with Transformers